src/components/NoEntityCard/NoEntityCard.tsx

```diff
@@ -11,7 +11,7 @@ const NoEntityCard = ({ title, message, actionButton }: Props) => (
     className="no-entity-card"
     data-testid={TestId.NO_ENTITY_CARD}
   >
-    <p className="u-no-padding--top">{message}</p>
+    <p className="u-no-margin--bottom">{message}</p>
```

Comment:
I think we still need to wrap it in `<p>` in order to add a little bit of padding on top. Also, the margin on the bottom should be removed in order to not push the button too far down.

With margin: 
![Screenshot from 2024-05-27 09-58-50](https://github.com/canonical/rebac-admin/assets/108150922/ec3e8c19-e2d7-4257-80c8-174222c3e470)

Without margin: 
![Screenshot from 2024-05-27 09-58-28](https://github.com/canonical/rebac-admin/assets/108150922/caec0a80-d564-4036-bd8a-1d27a6d8e102)

Reply:
Margin was removed: https://vanillaframework.io/docs/patterns/empty-state#no-content